pkg/plugins/hybrid/v1alpha/testdata/memcached-operator/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/example/memcached-operator
+
+go 1.16
+
+require (
+	k8s.io/apimachinery v0.20.2
+	k8s.io/client-go v0.20.2
+	sigs.k8s.io/controller-runtime v0.8.3
+)
```

Comment:
Do we need to add the helm-operator-plugins repo here as well?

Reply:
yeah, we are using that for loading the watches in `main.go` (https://github.com/operator-framework/helm-operator-plugins/pull/97/files#diff-770b9355255b5219a1408c302fc74857b3d737e7c71a67755ed97abfa6ada8c4R285). 
If we want to skip this, we can remove scaffolding of code which loads `watch.yaml`. And maybe add a note in while scaffolding?

Reply:
I think we should keep that code and add this repo as a dep